README.md

```diff
@@ -25,17 +24,41 @@ See `defaults/main.yml`
 
 ## Development and testing
 
-This role uses [Molecule](https://molecule.readthedocs.io/en/latest/#) and [Docker](https://www.docker.com/) for tests. 
+This role uses [Molecule](https://molecule.readthedocs.io/en/latest/#) and [Docker](https://www.docker.com/) for tests.
 
 After installing [Docker](https://www.docker.com/):
-```
+
+Clone the repository into a package named `marvel-nccr.aiida` (the folder must be named the same as the Ansible Galaxy name)
+
+```bash
 git clone https://github.com/marvel-nccr/ansible-role-aiida marvel-nccr.aiida
-# Note: folder name marvel-nccr.aiida is required for running tests
 cd marvel-nccr.aiida
+```
+
+Then run:
+
+```bash
 pip install -r requirements.txt  # Installs molecule
 molecule test  # runs tests
 ```
 
+or use tox (see `tox.ini`).
```

Comment:
Should we perhaps recommend tox instead?

Reply:
I'm going to leave both options here, because even though I'm on the `tox` train, I don't want to force people to use it if they would rather not